autoscaler/controllers/actions/tailsampling/config.go

```diff
@@ -0,0 +1,68 @@
+package sampling
+
+import (
+	"encoding/json"
+)
+
+type TailSamplingConfig struct {
+	Policies []Policy `json:"policies"`
+}
+type Policy struct {
```

Comment:
nit: might be worth mentioning the the policy is the opentelemetry collector tail_sampler processor policy, to give more context to future readers